### PR TITLE
Add popup menu

### PR DESCRIPTION
**Pull request by troyz**
I have added the popup menu into the demo application, the document have updated, please reference the section `Circle popup menu`.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ The implementation of `NADotAnnotation` places a red semi-transparent dot on the
 
 You can find a complete custom annotation example of multi-colored pins in [NAPinAnnotation.h](NAMapKit/NAPinAnnotation.h)/[.m](NAMapKit/NAPinAnnotation.m).
 
+####Circle popup menu
```
Comment (dblock) [inline README.md]: I think this should just be added to above, just like we say "you can find a complete custom annotation example in " maybe say "another example of ... can be found in". Make sure to link the source files.

**Comment by dblock** [comment]
Can you please update https://github.com/neilang/NAMapKit/blob/master/CHANGELOG.md, just say that you've added an example that does XYZ. Squash your commits if you don't mind, I'll merge this after, looks great. Thanks for hanging in here!

**Comment by troyz** [comment]
The `README.md` and `CHANGELOG.md` have updated, please correct me if there have any syntax error. :) @dblock

**Comment by dblock** [comment]
Merged via https://github.com/neilang/NAMapKit/commit/b3b7987076373cc1d71be0c003bf495dc4f90420, nice job!

**Comment by troyz** [comment]
Thanks!

**Comment by orta** [comment]
👍  closing as merged

**Pull request closed without merging**